DOCUMENTATION.rst

```diff
@@ -331,7 +331,7 @@ The constructor takes many keyword arguments that will affect its behavior.
 * ``vhost`` Default "/".
 * ``connect_timeout`` Default 5 seconds. Time before socket connection fails.
 * ``sock_opts`` Default None. Recommend at least ``{(socket.IPPROTO_TCP, socket.TCP_NODELAY) : 1}``
-* ``hearbeat`` Default None (disabled). If 0, broker assigned. If >0, negotiated with broker.
+* ``heartbeat`` Default None (disabled). If 0, broker assigned. If >0, negotiated with broker.
```

Comment:
@awestendorf: I just noticed that the description doesn't match the implementation. The implementation appears to be that if user passed `heartbeat=None`, then the heartbeat value is broker-assigned; if user passed `heartbeat=0`, then heartbeats would be disabled. The description is the reverse of what actually happens.

```
        if self.connection._heartbeat is None:
            self.connection._heartbeat = method_frame.args.read_short()
```